src/components/UserContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './UserContainer.css';
+import * as Types from '../Types';
+
+type UserContainerProps = {
+    displayName: Types.UserDisplay;
+    onClick: (displayName: Types.UserDisplay) => void;
+};
+
+type UserContainerState = {};
+
+export default class UserContainer extends Component<UserContainerProps, UserContainerState> {
```

Comment:
For these components that don't have state, is there a reason you chose to use `Component`, instead of `FC` like how you did for the `CustomButton`? If there's no state, I think I prefer `FC`

Reply:
Agree, and will change. It was probably just me being lazy and copy+pasting components :).